### ISSUE TITLE
Logging Does Not Capture Redeems with a Text Field as Redemptions

### ISSUE THREAD

**Issue by JCRouzer29** [Chatterino/chatterino2#5100]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

Any Twitch redeem of channel points which has a text field that is not left blank is saved in the log files as only the text within the redeem.  At times referencing which persons have done a redeem comes up but since information is required on their part we cannot rely on the logs in the event any are missed.

### Screenshots

<img width="607" alt="Logging Comparison - Redeems w Text" src="https://github.com/Chatterino/chatterino2/assets/13194274/af8a55e6-75ca-4a71-afed-0f8c425d250a">

### OS and Chatterino Version

Chatterino 2.4.6 on Windows 10 Version 22H2

**Comment by pajlada** [comment]
Thanks for the report! What would be your desired outcome here?
That the message logged, but that it includes the name of the redemption too?

**Comment by JCRouzer29** [comment]
Correct, including the name and cost of the redemption would be most helpful.  
Even if they are put on separate lines like how announcements are logged would work.
